ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -792,7 +793,7 @@ do
     if ! check_signed; then
         echo "You're updating to an unsigned images, you must override the verification using --force flag"
         FORCE_UPDATE="true"
-        if check_ota_case_enabled "OCP-30832" "OCP-27986" "OCP-24358" "OCP-69968"; then
+        if check_ota_case_enabled "OCP-30832" "OCP-27986" "OCP-24358" "OCP-69968" "OCP-56083"; then
```

Comment:
For `check_ota_case_enabled`, it only works with `ENABLE_OTA_TEST: <case-id>` case.